src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  width: fit-content;
+
+  &__star {
+    margin: 0 4px 0 0;
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
+}
```

Comment:
Stars active styles need to follow the [Stars task](https://github.com/mate-academy/layout_stars) requirements